lib/zcaps.js

```diff
@@ -51,18 +50,15 @@ exports.delegateCapability = async (
   if(caveat) {
     zcap.caveat = caveat;
   }
-  let expiryDate;
-  const now = Date.now();
   if(expires) {
-    expiryDate = new Date(expires);
-    if(expiryDate.getTime() < now) {
-      throw Error('The given expiration date has already passed.');
+    const expiresPattern =
+    new RegExp('^[1-9][0-9]{3}-(0[1-9]|1[0-2])-([0-2][0-9]|3[0-1])' +
+    'T([0-1][0-9]|2[0-3]):([0-5][0-9]):(([0-5][0-9])|60)(\\.[0-9]+)?' +
+    '(Z|((\\+|-)([0-1][0-9]|2[0-3]):([0-5][0-9])))?$');
```

Comment:
We don't usually do this kind of check at this code layer. @mattcollier do you have thoughts here?

Reply:
The indent starting on L55 is off.  :)  It is true that we don't do a lot of this sort of validation in nodejs APIs in favor of validating at the edges (HTTP APIs).  Could it be the case that improperly formatted dates being generated internally might be problematic and that issue is only discovered at some future date?

Aside from that consideration, it would be nice to have some canonical location for complicated regexes like this.  We have that sort of thing going on in the bedrock-validation module.  So, if it does turn out to be a good idea to validate this here, maybe the best approach would be to add an appropriate validator to bedrock-validation and use `validateInstance` to check it based on that schema.

Reply:
@dlongley Perhaps we don't need this check here then? Should I remove it?

Reply:
Let's just remove it.